velox/dwio/parquet/reader/PageReader.h

```diff
@@ -28,6 +30,8 @@
 
 namespace facebook::velox::parquet {
 
+using namespace dwio::common::compression;
```

Comment:
By convention, the header files do not use "using namespace". Can you please just inline these prefixes?